CMakeLists.txt

```diff
@@ -40,6 +39,7 @@ set(
 
 # Set environment hooks for default environment.
 if(WIN32)
+  set(BINARY_PATH_HOOK "${AMENT_PACKAGE_TEMPLATE_DIR}/environment_hook/path.${SHELL_EXT}")
```

Comment:
Isn't `path.bat` created in the same way as `path.sh` on Windows? Won't we have the same problem there?

Reply:
That is probably true, I was defaulting to not touching the windows implementation as much as possible :)

Reply:
Dropped Windows as well in 40d2176